### PR TITLE
SplitText has an option to remove empty strings or not

### PR DESCRIPTION
**Pull request by aveXcaesar**
### Please check if the PR fulfills these requirements
- [x] I made sure that my code builds
- [x] I merged the master into this branch before pushing
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

### What is the current behavior? (You can also link to an open issue here)
#119 

### What is the new behavior?
The SplitText ctors accepts a boolean to remove or don't remove empty strings.

### Does this PR introduce a breaking change? (check one with "x")
- [ ] Yes
- [ ] No

###  Other information

### CODE REVIEW

File: src/Yaapii.Atoms/Text/SplitText.cs [hunk 1]
```diff
@@ -40,56 +40,69 @@ public sealed class SplitText : IEnumerable<String>
...
         {
-            return new Filtered<String>(
+            var splitted =
+                new EnumerableOf<String>(
+                    new Regex(this._regex.AsString()).Split(this._origin.AsString()));
+
+            return 
+                this._remEmpty 
+                ? new Filtered<String>(
                     (str) => !String.IsNullOrEmpty(str),
```
Comment (Meerownymous) [inline src/Yaapii.Atoms/Text/SplitText.cs]: I think we should use ```String.IsNullOrWhiteSpace()```, don't we?

Comment (aveXcaesar) [reply src/Yaapii.Atoms/Text/SplitText.cs]: A snippet with a white space is a relevant information for me. I would suggest to keep white spaces

Comment (Meerownymous) [reply src/Yaapii.Atoms/Text/SplitText.cs]: I think in that case we should rename remEmpty parameter to remBlank.
When we need the blank ones we can make a filtered and filter by String.IsNullOrEmpty.
The use case with blanks is needed more often

Comment (aveXcaesar) [reply src/Yaapii.Atoms/Text/SplitText.cs]: Yes, I'm in.

**Pull request merged**